.github/workflows/manual-trigger-specific-test.yml

```diff
@@ -0,0 +1,31 @@
+# This is a basic workflow that is manually triggered
+
+name: Manual workflow - just for testing
+
+# Controls when the action will run. Workflow runs when manually triggered using the UI
+# or API.
+on:
+  workflow_dispatch:
+    # Inputs the workflow accepts.
+    input_command:
+      name:
+        description: 'CLI command you want to execute in the pipeline'
+
+jobs:
+  manual:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup test environment
+        uses: ./.github/actions/setup-env
+
+      - shell: bash
+        env:
+          ARCHIVE_EMAIL: ${{ secrets.ARCHIVE_EMAIL }}
+          ARCHIVE_PASSWORD: ${{ secrets.ARCHIVE_PASSWORD }}
+          PATRON_EMAIL: ${{ secrets.PATRON_EMAIL }}
+          PATRON_PASSWORD: ${{ secrets.PATRON_PASSWORD }}
+        run: |
+          ${{ github.event.inputs.input_command }}
```

Comment:
q? asking again if we need this manuall lever?

Reply:
for this one, we want to have a way to run specific tests in pipeline and compare it to the one that was ran locally